src/components/UserMessage.jsx

```diff
@@ -1,25 +1,31 @@
 import React, { useState, useEffect } from 'react';
+import { Link, useSearchParams } from 'react-router-dom';
 import user from '../img/user-img.png';
 import { UserResponseList } from './UserResponseList';
 import urlToJSON from './utils/urlToJSON';
-import { Link } from 'react-router-dom';
 import BackIcon from '../img/back.png';
+import createPathIdQuery from './utils/createPathIdQuery';
 
-export default function UserMessage({ setjsonPathMap, jsonPathMap, pathId }) {
+export default function UserMessage({ setjsonPathMap, jsonPathMap }) {
   const [userName, setuserName] = useState('Test-User >');
+
+  const [searchParams, setSearchParams] = useSearchParams();
+  const pathId = searchParams.get('pathId');
```

Comment:
From what I can tell in the previous implementation I'm guessing `pathId` is used as a search parameter. In `react-router-dom` v6 the new way to get search parameters is `useSearchParams` so I updated the code to use this to fetch the actual 'pathId' value